lib/modules/manager/sbt/extract.ts

```diff
@@ -289,6 +293,30 @@ const query = q.tree<Ctx>({
   postHandler: registryUrlHandler,
 });
 
+function extractUrl(line: string): string {
+  if (line.includes(',')) {
+    return line.substring(line.indexOf(':') + 1, line.indexOf(','));
+  } else {
+    return line.substring(line.indexOf(':') + 1);
+  }
+}
+
+export function extractProxyUrls(content: string): string[] {
+  const extractedProxyUrls: string[] = [];
+  logger.debug('Parsing proxy repository file');
+  for (const line of content.split(newlineRegex)) {
+    if (line.match(new RegExp('https?'))) {
+      const url = extractUrl(line);
```

Comment:
```suggestion
    if (line.match(regEx('https?'))) {
      const url = extractUrl(line);
```
Use the regex utils and this could be simplified by extracting the URL from the line directly by using regex groups